### ISSUE TITLE
How can I change "lib" compiler option to es2017?

### ISSUE THREAD

**Issue by b-dabrowski** [lukehaas/RunJS#270]
Hello,

I want to log values of my object

`Object.values(ProducerEmailSenderDomain);`

Everything works but I get this message and autocomplete doesn't show `values` prop in `Object`

![image](https://user-images.githubusercontent.com/21097396/123773459-f6c58580-d8cc-11eb-85d2-3fe6cd492fcd.png)

How can I change "lib compiler option to es2017" ?

**Comment by lukehaas** [comment]
Not quite sure why you're not seeing `values` in the list as it's working ok for me.

![Screenshot 2021-06-29 at 10 38 29](https://user-images.githubusercontent.com/741344/123775396-3fc60b80-d8c6-11eb-9de3-bf36195a12c4.png)

The lib compiler option is set to `esnext`. The tsconfig options are viewable inside the RunJS install directory.

**Comment by b-dabrowski** [comment]
`{
  "compilerOptions": {
    /* Basic Options */
    "target": "es2015",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
    "declaration": true,                   /* Generates corresponding '.d.ts' file. */
    "outDir": "./lib",                        /* Redirect output structure to the directory. */
    "esModuleInterop": true                   /* Enables emit interoperability between CommonJS and ES Modules via */
  }
}`

Interesting my `tsconfig.json` is set to `es2015` I changed it to `esnext` and still can't see `values` 

If I will reinstall RunJS, will my License Key still be valid?

**Comment by lukehaas** [comment]
Yes, the license key will always be valid.

**Comment by b-dabrowski** [comment]
Reinstalled the RunJS and still

![image](https://user-images.githubusercontent.com/21097396/123779886-eadcc200-d8d2-11eb-83ca-5f0cb319c3b8.png)

Can you give me the path to `tsconfig` so I will check it locally

Should I search it here?

![image](https://user-images.githubusercontent.com/21097396/123780125-26778c00-d8d3-11eb-9714-fd3f1c8b6c1a.png)

**Comment by lukehaas** [comment]
Once of these locations, depending on your OS:
Linux: ~/.config/runjs
Mac: ~/Library/Application\ Support/runjs
Windows: %APPDATA%\runjs

**Comment by b-dabrowski** [comment]
```
{
  compilerOptions: {
    allowJs: true,
    jsx: 'preserve',
    checkJs: true,
    allowSyntheticDefaultImports: true,
    allowNonTsExtensions: true,
    lib: ['dom', 'esnext'],
    target: 'es6',
    module: 'commonjs',
    strict: true,
    typeRoots: [
      './node_modules/@types',
      '/Applications/RunJS.app/Contents/Resources/packages/@types',
    ],
  },
  include: [],
  exclude: ['node_modules'],
};
```

Looks good interesting why I cant see the values prop :( I removed all npm packages from RunJS still nothing 🤔 

![image](https://user-images.githubusercontent.com/21097396/123781043-144a1d80-d8d4-11eb-85d8-d8d7fa631245.png)

**Comment by b-dabrowski** [comment]
I don't know if it will help but here is my mac info
![image](https://user-images.githubusercontent.com/21097396/123781744-e2858680-d8d4-11eb-8f0c-b9527b078980.png)

**Comment by lukehaas** [comment]
Really odd. I wonder if the tsconfig file is not reachable for some reason. 
Could you try making some arbitrary changes to it and seeing if they have an impact?

**Comment by b-dabrowski** [comment]
Can you tell me how to do that?

I removed RunJS and installed it via homebrew to check if it will help but I still see:

![image](https://user-images.githubusercontent.com/21097396/124430264-3e398f00-dd6f-11eb-9361-e8e52735c8f6.png)

**Comment by lukehaas** [comment]
The tsconfig file can be opened within an editor. While RunJS is open, please make some changes to the tsconfig that affect which diagnostics get displayed. This will confirm if the tsconfig file is reachable.
Documentation on tsconfig options can be found here: https://www.typescriptlang.org/tsconfig

**Comment by b-dabrowski** [comment]
```
{
   "compilerOptions":{
      "allowJs":true,
      "jsx":"preserve",
      "checkJs":true,
      "allowSyntheticDefaultImports":true,
      "allowNonTsExtensions":true,
      "lib":[
         "dom",
         "esnext"
      ],
      "target":"es6",
      "module":"commonjs",
      "strict":true,
      "typeRoots":[
         "./node_modules/@types",
         "/Applications/RunJS.app/Contents/Resources/packages/@types"
      ]
   },
   "include":[
      
   ],
   "exclude":[
      "node_modules"
   ]
}
```

When I change something in the `tsconfig.json` and open the RunJS my changes are overwritten :( 

As you can see lib is set to esnext :/

So why the RunJS on my machine can't see `Object.values` in autocomplete 🤔

**Comment by lukehaas** [comment]
You'll need to make the changes while RunJS is open because it will overwrite the file on start.

**Comment by lukehaas** [comment]
Also, could you tell me if you installed RunJS using the dmg or via Homebrew?

**Comment by b-dabrowski** [comment]
I changed `tsconfig.json` into empty configuration `{}` and no error occurs 😓 

A long time ago I installed it via dmg but I think I tried to install it via homebrew and the error was still there.

What is the best way to uninstall RunJS?

Should I install it via dmg or homebrew?

**Comment by lukehaas** [comment]
You can install RunJS either way, I was just looking for clues.

To uninstall completely, delete the app and delete the folder at ~/Library/Application\ Support/runjs

**Comment by b-dabrowski** [comment]
I removed everything and run `brew uninstall --cask runjs` then I run `brew install --cask runjs`

I started the app in the free option and I had not autocomplete modal and underscore with an error when I typed `Object.values({})`

Then I activated my licence and:

![image](https://user-images.githubusercontent.com/21097396/125575881-bb8578bc-3847-4405-a63e-fb8d16280dde.png)

What do you think can it be caused because of some global npm package installed?

Am I the only user with that issue 😅  ?

**Comment by lukehaas** [comment]
You're the only one so far who has had this issue.
I would suggest experimenting by uninstalling the homebrew version and trying it via the dmg.

One more idea, does your mac username have any spaces or unusual characters?

**Comment by b-dabrowski** [comment]
I will remove the homebrew version and install via dmg,

My username: Bartłomiej Dąbrowski (so, yes it has spaces and unusual chars)

**Comment by b-dabrowski** [comment]
I ran
1. `brew uninstall --cask runjs`
2. `rm -rf ~/Library/Application\ Support/runjs`

Then I installed RunJS from dmg and unfortunately still this (I am very sorry that I am bothering you with that issue 😢 )

![image](https://user-images.githubusercontent.com/21097396/125587323-8ecbbebc-5077-4dba-9cd5-2594052ff57d.png)

**Comment by lukehaas** [comment]
> I will remove the homebrew version and install via dmg,
> 
> My username: Bartłomiej Dąbrowski (so, yes it has spaces and unusual chars)

This is an interesting clue, I will do some investigating around this and get back to you.

**Comment by lukehaas** [comment]
@b-dabrowski it seems spaces etc in the file path are not an issue, so the cause of this issue is still unknown.

One question, are you able to install NPM packages through RunJS without issues?

**Comment by b-dabrowski** [comment]
NPM packages install correctly

![image](https://user-images.githubusercontent.com/21097396/125802587-2c31dbbd-f6be-49dc-abc7-e4e875cf599d.png)

And autocomplete works fine for the package:

![image](https://user-images.githubusercontent.com/21097396/125802690-a98523d2-e42e-4eea-981c-75591022aca1.png)

If you want we can arrange a meeting where I will show you how RunJS folder looks like after the installation but unfortunately today I don't have time for that 😢

**Comment by lukehaas** [comment]
@b-dabrowski I think I have a solution. I realised I have the `@types/node` package installed and that's why I'm seeing the `values` completion. So if you install that package it should fix it for you.

In the next release, I'll update this so that this package is included by default.

**Comment by b-dabrowski** [comment]
![image](https://user-images.githubusercontent.com/21097396/125909098-f5c8089c-42b2-4402-a2cd-72c945b4bb8d.png)

Yup! This is the solution 😄 

THX for your help ❤️

**Comment by Froskk** [comment]
I have run into this same issue. Installing @types/node fixes it on Mac, but not on my Windows machine. I have tried re-installing, removing/readding the @types module however I'm still left with no suggestions for `Object.entries1 etc. 

Also, similarly, `import path from 'path'` leaves a squiggly under 'path' saying `Cannot find module 'path' or its corresponding type declarations`

I have checked the tsconfig in %appdata% which has ```json
"typeRoots": [
      "./node_modules/@types",
      "C:\\Users\\rossc\\AppData\\Local\\Programs\\runjs\\resources\\packages\\@types"
    ]```
    which does seem to be correct. ./node_modules exists and has the @types module, and so does the local appdata folder
